make/migrations/postgresql/0005_1.9.0_schema.up.sql

```diff
@@ -0,0 +1,29 @@
+/*add webhook policy table*/
+create table webhook_policy (
+ id SERIAL NOT NULL,
+ name varchar(256),
+ project_id int NOT NULL,
+ enabled boolean NOT NULL DEFAULT true,
+ description text,
+ targets text,
+ hook_types text,
```

Comment:
I still find this column confusing, I believe you are using this column to filter events, right?  If the answers yes, why not rename it to `event_types`.  
Otherwise there's still no clear boundary for `hook` and `event`